packages/js/src/ui/components/elements/Preferences/Preferences.tsx

```diff
@@ -148,17 +148,14 @@ const PreferencesRow = (props: {
         <div
           class={style(
             'workflowLabelContainer',
-            'nt-flex nt-justify-between nt-flex-nowrap nt-self-stretch nt-cursor-pointer nt-items-center'
+            'nt-flex nt-justify-between nt-flex-nowrap nt-self-stretch nt-cursor-pointer nt-items-center nt-overflow-hidden'
           )}
           onClick={() => setIsOpen((prev) => !prev)}
           data-open={isOpen()}
         >
-          <div>
+          <div style={{ overflow: 'hidden' }}>
```

Comment:
If someone tried to override the default behavior via the appearance prop classnames, wouldn't this take precedence and prevent them from modifying behavior?

Reply:
Correct. Don't see any case for this to happen now though. We have no descriptor here so you can't override anyway. Can you come up with a good name for it?

Reply:
I don't think there's a particularly strong case either, but still good to keep it flexible. Here are some options:

1. workflowLabelSection
2. workflowLabelHeader
3. workflowLabelDisplay
4. workflowLabelPanel

Personally I would go with `workflowLabelHeader`.

Reply:
Done! Nice suggestion.